op-chain-ops/cmd/celo-migrate/non-ancients.go

```diff
@@ -89,7 +90,7 @@ func migrateNonAncientsDb(oldDbPath, newDbPath string, numAncients, batchSize ui
 	}
 
 	if numAncients > 0 {
```

Comment:
Do we need to change that condition as well?

Reply:
Well spotted